neqo-client/src/main.rs

```diff
@@ -1018,6 +1018,7 @@ fn main() -> Res<()> {
                 args.use_old_http = true;
                 args.key_update = true;
             }
+            "v2" => {}
```

Comment:
The v2 test also runs on HTTP/0.9. Do you need to set `use_old_http` here?

Reply:
Yes, exactly right. Thank you @marten-seemann. See https://github.com/mozilla/neqo/pull/1570/commits/61ed61ebd66320ab40f245afc1fa36bbbebd8784.

![image](https://github.com/mozilla/neqo/assets/7047859/bf35d72b-8f4c-4b10-9627-8935a6bb1ccf)

Reply:
Exciting!